src/main/java/com/cloudant/client/org/lightcouch/Param.java

```diff
@@ -50,25 +51,19 @@ public String toURLEncodedString() {
     }
 
     private static String encodeQueryParameter(String in) {
+        // As described above, we need to use the Java URI class to encode
```

Comment:
Comment doesn't make sense to me. Did you copy this method from sync-android? Either way, this method is of little use, it's probably better to change the call sites to call `HierarchicalUriComponents.encodeUriComponent` directly.